packages/sveltekit/src/client/browserTracingIntegration.ts

```diff
@@ -1,14 +1,164 @@
-import { BrowserTracing as OriginalBrowserTracing } from '@sentry/svelte';
+import { navigating, page } from '$app/stores';
+import { SEMANTIC_ATTRIBUTE_SENTRY_ORIGIN, SEMANTIC_ATTRIBUTE_SENTRY_SOURCE } from '@sentry/core';
+import {
+  BrowserTracing as OriginalBrowserTracing,
+  WINDOW,
+  browserTracingIntegration as originalBrowserTracingIntegration,
+  getActiveSpan,
+  startBrowserTracingNavigationSpan,
+  startBrowserTracingPageLoadSpan,
+  startInactiveSpan,
+} from '@sentry/svelte';
+import type { Client, Integration, Span } from '@sentry/types';
 import { svelteKitRoutingInstrumentation } from './router';
 
 /**
  * A custom BrowserTracing integration for Sveltekit.
+ *
+ * @deprecated use `browserTracingIntegration()` instead. The new `browserTracingIntegration()`
+ * includes SvelteKit-specific routing instrumentation out of the box. Therefore there's no need
+ * to pass in `svelteKitRoutingInstrumentation` anymore.
  */
 export class BrowserTracing extends OriginalBrowserTracing {
   public constructor(options?: ConstructorParameters<typeof OriginalBrowserTracing>[0]) {
     super({
+      // eslint-disable-next-line deprecation/deprecation
       routingInstrumentation: svelteKitRoutingInstrumentation,
       ...options,
     });
   }
 }
+
+/**
+ * A custom `BrowserTracing` integration for SvelteKit.
+ *
+ * @param options
```

Comment:
```suggestion
```

I'd remove this because this messes with the types you otherwise get by default 😅